src/pages/Overview/BalanceLinks.tsx

```diff
@@ -6,12 +6,14 @@ import { ButtonPrimaryInvert, Separator } from '@polkadot-cloud/react';
 import { useTranslation } from 'react-i18next';
 import { useApi } from 'contexts/Api';
 import { useConnect } from 'contexts/Connect';
+import { NetworkNameEnum } from 'types';
 import { MoreWrapper } from './Wrappers';
 
 export const BalanceLinks = () => {
   const { t } = useTranslation('pages');
   const { name } = useApi().network;
   const { activeAccount } = useConnect();
+  const networkNameDomainPart = name.toLocaleLowerCase().replaceAll(' ', '');
```

Comment:
How are we sure that this transformation always gives as a proper subscan domain? Is this subscan's official recommendation?

Reply:
network names are controlled by us and the only network that this button is enabled on is mainnet, so it does result in correct url https://alephzero.subscan.io/account/. Perhaps I should just hardcode it in here? I don't think it matters

Reply:
done